### PR TITLE
Fix docs about how to install development version

### PR DESCRIPTION
**Pull request by justinmayer**
The current version of Pip can't do editable installs without setup.py, so at present running `pip install -e .` will fail.

### CODE REVIEW

**Comment by justinmayer** [comment]
If at some point a **Contributing** file and/or README section is added to this project, it should probably note that at present the best way to install Django-Tailwind in editable mode is to add the following line to one's Django project `pyproject.toml` file, pointing via relative path to a local Git clone:
```toml
django-tailwind = { path = "../relative/path/to/django-tailwind", develop = true }
```

**Comment by justinmayer** [comment]
Hey Tim. Any thoughts on this pull request? Happy to clarify if there's anything unclear. 😁

File: README.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 ## Quick start
 
-1. Install the python package django-tailwind from pip
+1. Install the `django-tailwind` package via Pip:
 
-   `pip install django-tailwind`
+   `python -m pip install django-tailwind`
```
Comment (timonweb) [inline README.md:7]: Why did you change this? Isn't it more verbose?

File: README.md [hunk 2]
```diff
@@ -2,11 +2,13 @@
...
 
-   Alternatively, you can download or clone this repo and run `pip install -e .`.
+   Alternatively, you can install the latest development version via:
+
+   `python -m pip install git+https://github.com/timonweb/django-tailwind.git`
```
Comment (timonweb) [inline README.md:11]: Same here, can't it be done without `python -m`?

**Comment by timonweb** [comment]
Hi Justin, thanks for the PR, looks good to me, but left some comments.

File: README.md (hunk 1 above)
Comment (justinmayer) [reply README.md:7]: Apologies for not explaining. The short answer is because (1) [it eliminates confusion regarding which Python interpreter will be invoked](https://github.com/pypa/pip/issues/3164#issue-109993120), and (2) [Python itself switched to this recommended invocation](https://docs.python.org/3/installing/index.html#basic-usage).

File: README.md (hunk 2 above)
Comment (justinmayer) [reply README.md:11]: See also:
https://snarky.ca/why-you-should-use-python-m-pip/
https://adamj.eu/tech/2020/02/25/use-python-m-pip-everywhere/

File: README.md (hunk 1 above)
Comment (timonweb) [reply README.md:7]: Thanks for the links, although I don't like it and never use it, I'd rather be in sync with Python docs.

Comment (justinmayer) [reply README.md:7]: I don't use the `python -m pip` syntax either, but then again I only invoke Pip from within virtual environments, and thus it's always clear which interpreter I am using. Less-experienced folks are more likely to be bitten by bare `pip` invocations, which is why I believe this change in the docs is the right way to go. 👍
